app/src/main/java/com/liato/bankdroid/lockpattern/LockPatternUtils.java

```diff
@@ -143,7 +144,11 @@ public static String patternToString(List<LockPatternView.Cell> pattern) {
             LockPatternView.Cell cell = pattern.get(i);
             res[i] = (byte) (cell.getRow() * 3 + cell.getColumn());
         }
-        return new String(res);
+        try {
```

Comment:
Do you mind moving this into a private method to keep this method more readable? as you did with `toUtf8Bytes`in `SimpleCrypto`?